### DISCUSSION TITLE
Rest API limits result for x number of items

### DISCUSSION THREAD

**Discussion by Fursel** [grafana/grafana-infinity-datasource#517] [Q&A]
Hello,

Is there a way of configuring Inifnity plugin to iterate though URLs until some number or if no more results are returned ?

I am trying to pool data to Graphana with Rest API and it is being limited for 100 items to display but parameter can be used at the end of the URL to list 100 more after id and so on. 

Wonder if it is possible to have it somehow done instead creating multiple queries

**Comment by svet-b** [comment]
Automatic pagination isn't currently supported, but you can potentially DIY it by doing something like https://github.com/yesoreyeram/grafana-infinity-datasource/discussions/488#discussioncomment-4458422
